docs/src/whatsnew/3.9.rst

```diff
@@ -1,21 +1,26 @@
 .. include:: ../common_links.inc
 
-|iris_version| |build_date| [unreleased]
-****************************************
+v3.9 (03 Apr 2024) [release candidate]
+**************************************
 
 This document explains the changes made to Iris for this release
 (:doc:`View all changes <index>`.)
 
 
-.. dropdown:: |iris_version| Release Highlights
+.. dropdown:: v3.9 Release Highlights
    :color: primary
    :icon: info
    :animate: fade-in
    :open:
 
-   The highlights for this major/minor release of Iris include:
+   This is a small release to make two important changes available as soon as
+   possible:
 
-   * N/A
+   * The :mod:`iris.experimental.geovista` module.
+   * Removal of fill value collision warnings in NetCDF saving, which
+     significantly improves Iris' stability when parallel processing.
```

Comment:
Not sure stability is the right term?
```suggestion
     significantly improves Iris' performance when parallel processing.
```

Reply:
OK I suppose this is still performance, yeah. I was thinking about the unpredictability and this perhaps misled me!